src/core/qgsmultirenderchecker.cpp

```diff
@@ -119,6 +119,9 @@ QgsLayoutChecker::QgsLayoutChecker( const QString &testName, QgsLayout *layout )
 
 bool QgsLayoutChecker::testLayout( QString &checkedReport, int page, int pixelDiff, bool createReferenceImage )
 {
+#ifdef QT_NO_PRINTER
+  return false;
+#else
```

Comment:
Where is QPrinter used here?

Reply:
The class QgsLayoutExporter is not defined at all with QT_NO_PRINTER, see the qgslayoutexporter.hpp

Reply:
Hm, ok. That seems odd to me, when only a few methods in QgsLayoutExporter use QPrinter. All the image and svg based exports should still work without issue.